### PR TITLE
vinaigrette

### PR DESCRIPTION
**Pull request by ntgalili**

### CODE REVIEW

File: Desktop/exellenteam_python/vingert.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import random
+import datetime
+
```
Comment (itayday) [inline Desktop/exellenteam_python/vingert.py:3]: Please divide your code into functions.
When needed, call them inside a main function.
Remember to document.

File: Desktop/exellenteam_python/vingert.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+strdate1=input("enter a date (YYYY-MM-DD)")
```
Comment (talbarfm) [inline Desktop/exellenteam_python/vingert.py:4]: dont add the types in the variable names

Comment (talbarfm) [inline Desktop/exellenteam_python/vingert.py:4]: dont add numbers in variable names unless its for a good reason - in here, it makes way more sense to call the variables start_date and end_date - they have different meanings, they are not just two dates

Comment (talbarfm) [reply Desktop/exellenteam_python/vingert.py:4]: change all your variable names to ones with meaning

File: Desktop/exellenteam_python/vingert.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+epoch1=date1.timestamp()
+epoch2=date2.timestamp()
+
+rand_epoch=random.randint(epoch1,epoch2)
+
+new_date = datetime.datetime.fromtimestamp(rand_epoch)
+week_day= new_date.weekday()
+
+print(new_date.strftime('%Y-%m-%d'))
```
Comment (talbarfm) [inline Desktop/exellenteam_python/vingert.py:18]: https://www.freecodecamp.org/news/if-name-main-python-example/ 
Read this - add this when you add a main function

File: Desktop/exellenteam_python/vingert.py (hunk 1 above)
Comment (talbarfm) [reply Desktop/exellenteam_python/vingert.py:3]: Capture
This is how it should look like

**Review by talbarfm — CHANGES_REQUESTED** [review]

**Pull request closed without merging**